### PR TITLE
test: keep spaces

### PR DESCRIPTION
**Pull request by crazyair**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    test       |
| 🇨🇳 Chinese |    test       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by li-jia-nan** [comment]
为啥要改这个？

**Comment by crazyair** [comment]
> 为啥要改这个？

看 react 16 & 18 差异，就是会有多空行的问题，不会有空格的差异，看下这个 test 能跑成功吗

File: tests/setupAfterEnv.ts [hunk 1]
```diff
@@ -49,9 +51,8 @@ function formatHTML(nodes: any) {
   });
 
   const filtered = htmlContent
-    .split(/[\n\r]+/)
+    .split('\n')
```
Comment (MadCcc) [inline tests/setupAfterEnv.ts:54]: 这行不该改吧

Comment (crazyair) [reply tests/setupAfterEnv.ts:54]: react 16 18 的区别，就是只有 \n 换行，没有 \r
```html
<div>
1111

</div>
```

Comment (crazyair) [reply tests/setupAfterEnv.ts:54]: 被 `pretty-format` 转成 html 字符串的空行，只是 `\n`

**Comment by zombieJ** [comment]
神奇，之前这个多余的 ` ` 在 16 里一定会挂掉，居然现在不挂了。

**Review by zombieJ — DISMISSED** [review]

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**